aix/README.md

```diff
@@ -0,0 +1,17 @@
+# AixLib
+
+This [W3ID](https://w3id.org) provides a persistent URI namespace for the Ontology representing the [Modelica AixLib](https://github.com/RWTH-EBC/AixLib) library ontology.
+
+
+## Contact
+
+This space is administered by:
+
+**Elisabeth Eckstädt**
+_R&D Engineer_
+[Fraunhofer IIS/EAS](hhttps://www.eas.iis.fraunhofer.de/)
+Dresden, Germany
+<elisabeth.eckstaedt@eas.iis.fraunhofer.de>
+GitHub: [ElisEck](https://github.com/ElisEck)
+Linkedin: [Elisabeth Eckstädt](https://www.linkedin.com/in/elisabeth-eckst%C3%A4dt-18689115a)
+ORCID: [0000-0003-4396-5307](https://orcid.org/0000-0003-4396-5307)
```

Comment:
**Elisabeth Eckstädt** <br>
_R&D Engineer_ <br>
[Fraunhofer IIS/EAS](hhttps://www.eas.iis.fraunhofer.de/) <br>
Dresden, Germany <br>
<elisabeth.eckstaedt@eas.iis.fraunhofer.de> <br>
GitHub: [ElisEck](https://github.com/ElisEck) <br>
Linkedin: [Elisabeth Eckstädt](https://www.linkedin.com/in/elisabeth-eckst%C3%A4dt-18689115a) <br>
ORCID: [0000-0003-4396-5307](https://orcid.org/0000-0003-4396-5307)